Assets/Mirror/Runtime/NetworkManagerHUD.cs

```diff
@@ -31,9 +37,32 @@ public class NetworkManagerHUD : MonoBehaviour
         /// </summary>
         public int offsetY;
 
+        /// <summary>
+        /// Outputs the current devices IP to gui input, useful if your are host/server.
+        /// Only use the WAN check when necessary, as it uses an external service.
+        /// For further connection information, visit: https://mirror-networking.com/docs/Articles/FAQ.html#how-to-connect
+        /// </summary>
+        [Tooltip("localhost for games on same device, LAN for different devices same network, WAN for external connections. See summary for more information.")]
+        [SerializeField]
+        private IPType _IPType = IPType.localhost;
```

Comment:
in C# everything is 'private' by default. please avoid unnecessary 'private' declarations